import-scripts/subset-impact-data.sh

```diff
@@ -55,6 +55,14 @@ if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
 fi
 echo -e "\tPORTAL_SCRIPTS_DIRECTORY="$PORTAL_SCRIPTS_DIRECTORY
 
+# check for presence of new filename pattens (not yet supported) and fail if present
+
+$PYTHON_BINARY $PORTAL_SCRIPTS_DIRECTORY/study_directory_uses_updated_filenames.py -d $INPUT_DIRECTORY
+if [ $? -eq 0 ] ; then
```

Comment:
checking for -eq 0 (which is technically working right now for the main test, but we still have a nonzero exit code for a failed test in one of the other checks) 

to keep it simple, all failed tests (meaning new files present, or anything else) should stick to a non-zero exit code and the check can be for -ne 0